dig/dig.go

```diff
@@ -120,6 +127,13 @@ func (g *Graph) Resolve(obj interface{}) (err error) {
 	return nil
 }
 
+// MustResolve calls Resolve and panics if an error is encountered
+func (g *Graph) MustResolve(obj interface{}) {
```

Comment:
The "don't" panic should read more like "don't only panic".

We clearly have an API if users want to validate each error and work around it. Which is also a great argument to keep the API as is to illustrate the "default" behaviour